static/Cost/300_Optimization_Data_Collection/Code/Optimization_Data_Collector.yaml

```diff
@@ -265,7 +265,7 @@ Resources:
     Condition: DeployEcsChargebackModule
     Properties:
       TemplateURL: "https://aws-well-architected-labs.s3.us-west-2.amazonaws.com/Cost/Labs/300_Optimization_Data_Collection/ecs_data.yaml"
-      TimeoutInMinutes: 2
+      TimeoutInMinutes: 5
```

Comment:
Do you think we should make the Lambda timeouts longer? to avoid people with more resources getting stuck?